src/components/data-management/SamplesTable.jsx

```diff
@@ -72,7 +72,24 @@ const SamplesTable = forwardRef((props, ref) => {
   const [samplesLoaded, setSamplesLoaded] = useState(false);
 
   const initialTableColumns = useMemo(() => {
-    if (_.isNil(selectedTech)) return [];
```

Comment:
This was the culprit, instead of returning an empty array, return the first columns